frost-msg-spec/msg_spec.tex

```diff
@@ -0,0 +1,186 @@
+% Created 2021-04-15 Thu 21:55
```

Comment:
Again, let's do documentation using the Rust doc format that Zebra uses.

Reply:
I generated all these file formats from a single .org file when I didn't know which one we were going with, they have the same contents and can be deleted.